dashboard/client/src/components/LogView/LogVirtualView.tsx

```diff
@@ -93,6 +100,70 @@ const useStyles = makeStyles((theme) =>
   }),
 );
 
+type LogLineDetailDialogProps = {
+  formattedLogLine: string;
+  onClose: () => void;
+};
+
+const LogLineDetailDialog = ({
+  formattedLogLine,
+  onClose,
+}: LogLineDetailDialogProps) => {
+  return (
+    <DialogWithTitle
+      title="Log line details"
+      handleClose={() => {
+        onClose();
+      }}
```

Comment:
```suggestion
      handleClose={onClose}
```

nit: unnecessary additional function. Also avoids re-rendering the component since the function identifier didn't change.